src/games/rcll/production.clp

```diff
@@ -421,6 +421,16 @@
 	           (cs-retrieved (eq ?cs-op RETRIEVE_CAP)))
 )
 
+(defrule production-cs-process-retrieved-cap
+  "Award points for retrieved cap"
+  ?m <- (machine (name ?n) (mtype CS) (state PROCESSED) (team ?team) (cs-operation RETRIEVE_CAP))
+  =>
```

Comment:
This still won't work, for similar reasons as before: If the machine fact changes but the conditions are still satisfied, the rule may fire twice. I expect this will actually happen because the machine's `mps-busy` will change from `TRUE` to `FALSE`, which will still satisfy the LHS of this rule.

Reply:
Thanks for pointing this out. Indeed, I had encountered this while testing the workpiece tracking but had not pushed the fix yet to this branch. 
Thanks, please have another look!